recce/cli.py

```diff
@@ -184,6 +184,131 @@ def version():
     print(__version__)
 
 
+@cli.command(cls=TrackCommand)
+@add_options(recce_dbt_artifact_dir_options)
```

Comment:
Please add

```
@add_options(dbt_related_options)
```

So that you can have `--target`, `--profile`, `--project-dir`, and `-profiles-dir`